sys/suit/v4/cbor.c

```diff
@@ -88,13 +88,13 @@ int suit_cbor_get_int(const CborValue *it, int *out)
 int suit_cbor_get_string(const CborValue *it, const uint8_t **buf, size_t *len)
 {
     if (!(cbor_value_is_text_string(it) || cbor_value_is_byte_string(it) || cbor_value_is_length_known(it))) {
-        return -1;
+        return SUIT_ERR_INVALID_MANIFEST;
```

Comment:
while this might be the same value, at this level, there shouldn't be any manifest define in the code. It is just a cbor helper.

Reply:
But this is now consistent with other `suit_cbor_get_<whatever type>` functions.

Reply:
ok